colorbrewer/README.md

```diff
@@ -0,0 +1,19 @@
+# cljsjs/textures
```

Comment:
Whoops! Will fix when I get the chance. Thanks man. 

Sent from my iPhone

> On 31 Mar 2016, at 17:48, Juho Teperi <notifications@github.com> wrote:
> 
> In colorbrewer/README.md:
> 
> > @@ -0,0 +1,19 @@
> > +# cljsjs/textures
> The readme still refers other package in this header and in dependency vector below.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>